docs/9_0/core/ajaxRendering.md

```diff
@@ -148,4 +148,17 @@ This can be easily archived by attaching `p:autoUpdate` to the component:
 
 ### Skipping auto update
 
-You can also skip all auto-updates by setting _ignoreAutoUpdate_ to `true` on the trigger component (like `p:ajax`)
\ No newline at end of file
+You can also skip all auto-updates by setting _ignoreAutoUpdate_ to `true` on the trigger component (like `p:ajax`)
```

Comment:
i wouldnt add this in "ajax rendering", thats not related to AJAX directly
of course refresh and destroy is only called on AJAX but postConstruct is always called

Reply:
Any preference on where to put it in the docs?  I couldn't quite figure out the best place.

Reply:
I am actually thinking under core we need a whole new page title "Widgets".  I think there is a lot of info about Widgets we can add there.